cluster/src/main/java/io/scalecube/cluster/gossip/IGossipProtocol.java

```diff
@@ -12,10 +15,19 @@
  */
 public interface IGossipProtocol {
 
```

Comment:
Following ``start()`` , ``stop()``  methods doesn't need live here, they can be moved to implementation class. Same applies to ``IFailureDetector`` start/stop.